PhoneAuth/FirebasePhoneAuthUI/FUICodeField.m

```diff
@@ -25,49 +25,37 @@
 
 @interface FUICodeField ()
 
-@property (nonatomic, retain, readonly) UIView *inputField;
-
-@property (weak, nonatomic) IBOutlet UILabel *digits;
-
-@property (nonatomic, readonly) IBInspectable NSString *placeholder;
+@property (nonatomic, readonly) IBInspectable NSString *placeholderChar;
 
 @end
 
 @implementation FUICodeField
 
 - (instancetype)initWithFrame:(CGRect)frame {
   if (self = [super initWithFrame:frame]){
-    [self setUpFromNib];
+    [self commonInit];
   }
   return self;
 }
 
 - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
   if (self = [super initWithCoder:aDecoder]){
-    [self setUpFromNib];
+    [self commonInit];
   }
   return self;
 }
 
-- (void)setUpFromNib {
-  NSBundle *bundle = [FUIAuthUtils bundleNamed:FUIPhoneAuthBundleName];
-  UINib *nib = [UINib nibWithNibName:NSStringFromClass([self class]) bundle:bundle];
-
-  _inputField = [nib instantiateWithOwner:self options:nil][0];
-  self.inputField.frame = [self bounds];
-  self.inputField.autoresizingMask =
-      UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-  self.inputField.userInteractionEnabled = YES;
-
-  if (@available(iOS 12.0, *)) {
-    if ([self.inputField respondsToSelector:@selector(setTextContentType:)]) {
-      id<UITextInputTraits> inputField = (id<UITextInputTraits>)self.inputField;
-      inputField.textContentType = UITextContentTypeOneTimeCode;
-    }
-  }
-
+- (void)commonInit {
   // Initialization code
   _codeEntry = [NSMutableString string];
+  self.backgroundColor = UIColor.clearColor;
+  self.tintColor = UIColor.clearColor;
+  self.font = [UIFont fontWithName:@"Courier" size:40];
+  self.textAlignment = NSTextAlignmentLeft;
+  UIView *paddingView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 20, self.bounds.size.height)];
+  self.leftView = paddingView;
+  self.leftViewMode = UITextFieldViewModeAlways;
+  self.textContentType = UITextContentTypeOneTimeCode;
```

Comment:
Please keep the `if (@available(iOS 12.0, *))` check around this line since it's only available on iOS 12 and higher.

Reply:
I'll accept the PR as is and add this line manually since I want to get it into the next release. Thanks @akivab!